generator/src/main/java/com/scottlogic/deg/generator/generation/string/IsinStringGenerator.java

```diff
@@ -146,7 +146,9 @@ public boolean match(String subject) {
                     generateAllInvalidCheckDigitIsins()));
         }
         final List<Iterable<String>> countryCodeIterables = getAllCountryIsinGeneratorsAsStream()
-            .map(generator -> wrapIterableWithProjectionAndFilter(generator.generateAllValues()))
```

Comment:
We should be able to delete this `wrapIterableWithProjectionAndFilter` now I think?

Reply:
it's still used elsewhere